src/lib/common/globals.cpp

```diff
@@ -188,6 +189,69 @@ void orionInit
 
 /* ****************************************************************************
 *
+* mongoInit -
+*/
+void mongoInit
```

Comment:
I chose globals.cpp/h due to orionInit() was already there. However, you are right that probably makes more sense as part of mongoBackend. To be moved.